randovania/generator/item_pool/pickup_creator.py

```diff
@@ -83,6 +84,8 @@ def create_ammo_expansion(ammo: Ammo,
     if resource_database.item_percentage is not None:
         resources.append((resource_database.item_percentage, 1))
 
+    item_database = default_database.item_database_for_game(resource_database.game_enum)
```

Comment:
Definitely should get the category from arguments. Maybe the `Ammo` class should have it?